apps/omg_api/lib/block_queue/core.ex

```diff
@@ -126,6 +126,10 @@ defmodule OMG.API.BlockQueue.Core do
     end
   end
 
+  def enqueue_block(state) do
```

Comment:
unnecessary function, handle_cast should send what the got to Core

Reply:
```
 def handle_cast({:enqueue_block, %Block{transactions: []}}, %Core{} = state) do
      state1 = Core.enqueue_block(state)
      {:noreply, state1}
    end

    def handle_cast({:enqueue_block, %Block{number: block_number, hash: block_hash}}, %Core{} = state) do
      state1 = Core.enqueue_block(state, block_hash, block_number)

      _ =
        Logger.info(fn ->
          "Enqueuing block num '#{inspect(block_number)}', hash '#{inspect(Base.encode16(block_hash))}'"
        end)

      submit_blocks(state1)
      {:noreply, state1}
    end
```

The extra handle_cast is to prevent to run following code:
```
 _ =
        Logger.info(fn ->
          "Enqueuing block num '#{inspect(block_number)}', hash '#{inspect(Base.encode16(block_hash))}'"
        end)

      submit_blocks(state1)
```